frontend/src/components/Map/Zoom.tsx

```diff
@@ -19,11 +18,17 @@ const zoomButtomStyle = css`
 `;
 
 interface Props {
+  zoom: number;
+  defaultZoom: number;
   zoomStep: number;
+  onZoomChanged: (zoom: number) => void;
+  onFitScreen: () => void;
 }
 
-export function Zoom({ zoomStep = 1 }: Props) {
-  const [zoom, setZoom] = useAtom(zoomAtom);
+export function Zoom(props: Props) {
```

Comment:
[nit] Maybe the `Zoom` component could now be renamed to something more generic, like `MapControls`? 🤷